src/roster/templates/roster/student-details.html

```diff
@@ -0,0 +1,203 @@
+                        </th>
+                        <td>0.5 <span aria-hidden="true">hrs</span><span class="sr-only">hours</span> (2 visits)</td>
+                        <td>Wed, Feb 9</td>
+                        <td class="d-sm-down-none">consectetur, adipiscing, elit</td>
+                        <td class="d-sm-down-none">A lot / I will try to speak up more</td>
+                        <td class="d-md-down-none">
+                            <span class="muted">(What most suprised you?)</span>
+                            That words have power
+                        </td>
+                        <td class="d-md-down-none">
+                            Middle school
+                        </td>
+                    </tr>
+                    <tr>
+                        <th scope="row">
+                            <a href="#">Amelia Earhart: Meet Amazing Americans</a>
+                            <span class="datatable-mark"><span class="sr-only">Assigned</span></span>
+                            <div class="muted">Library of Congress</div>
+                        </th>
+                        <td>0.3 <span aria-hidden="true">hrs</span><span class="sr-only">hours</span> (1 visit)</td>
+                        <td>Tue, Feb 8</td>
+                        <td class="d-sm-down-none">Lorem, ipsum, dolor</td>
+                        <td class="d-sm-down-none">Nothing / I will try to speak up more</td>
+                        <td class="d-md-down-none">
+                        </td>
+                        <td class="d-md-down-none">
+                            Middle school
+                            <span class="muted">(Challenge 3)</span>
+                        </td>
+                    </tr>
+                    <tr>
+                        <th scope="row">
+                            <a href="#">All About Coyotes</a>
+                            <div class="muted">Lauren Poniatowski</div>
+                        </th>
+                        <td>0.1 <span aria-hidden="true">hrs</span><span class="sr-only">hours</span> (1 visit)</td>
+                        <td>Tue, Feb 8</td>
+                        <td class="d-sm-down-none">Lorem, ipsum, dolor</td>
+                        <td class="d-sm-down-none">A little</td>
+                        <td class="d-md-down-none">
+                        </td>
+                        <td class="d-md-down-none">
+                            High school
+                        </td>
+                    </tr>
+                </tbody>
+            </table>
+        </div>
+    </div>
+</main>
+{# Unsure if the tour or tips are available on the student detail page #}
+{# Leaving in place for now #}
+{% for tour in tours %}
+{% include "shared/partial/popover_tour.html" with tour_name=tour.name tour_robust=tour.robust %}
+{% endfor %}
+{% if tip_name %}
+{% include "shared/partial/tooltip_tip.html" %}
+{% endif %}
```

Comment:
Not needed as this is already included in the [base.html](https://github.com/cast-org/clusive/blob/40837425056fcf1f65420ddb4067280031730b76/src/shared/templates/shared/base.html#L74-L79) template.  This template builds on top of the base.

Reply:
Just double checked with MacOS and Win FF with the function call - the Masonry layout is working fine for me.  Chrome, Safari, and Edge are also working.  Checked with reduced motion/animation OS settings also.

Could it be a browser cache issue?

Reply:
@mbrambilla  Just now I added the script to the template for the first time today, and the layout of the details page is fine in FF.  I've pushed the change here.

@cindyli The student details template has one more change.  Could you merge it with your work and see if the layout is okay for you, too?  Thanks.

Reply:
@klown Merged this branch into my working branch. Tested with FF, Chrome and Safari on Mac. The layout looks fine on them.

Reply:
🎊